src/main/java/hlf/java/rest/client/controller/ChaincodeOperationsController.java

```diff
@@ -53,6 +53,20 @@ public ResponseEntity<String> performChaincodeOperation(
     return new ResponseEntity<>(operationResponse, HttpStatus.OK);
   }
 
+  @GetMapping(value = "/version")
+  @MetricsTrackedEndpoint(name = "Get Version", method = "GET", uri = "/chaincode/version")
+  public ResponseEntity<String> getCurrentSequence(
```

Comment:
[nitpick] Since we are getting version how about naming it to `getCurrentVersion`

Reply:
Nice catch, Corrected.